Makefile

```diff
@@ -12,6 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
+ENVTEST_K8S_VERSION = 1.29.1
```

Comment:
There's a new way to do this, copy https://github.com/openshift/cluster-cloud-controller-manager-operator/commit/a27aaee1f264dd752266b9ef4f9d1590d8cea4be please

Reply:
oop ok - taking a look

Reply:
Sweet - updated!